tools/run_tests/performance/scenario_config.py

```diff
@@ -560,25 +586,28 @@ def scenarios(self):
           secure=secure,
           categories=smoketest_categories)
 
+      # unconstrained_client='async' is intended (client uses goroutines)
       yield _ping_pong_scenario(
           'go_protobuf_sync_unary_qps_unconstrained_%s' % secstr, rpc_type='UNARY',
           client_type='SYNC_CLIENT', server_type='SYNC_SERVER',
-          use_unconstrained_client=True,
+          unconstrained_client='async',
```

Comment:
Yes, go uses goroutines and not real threads, so using the async values for unconstrained seems to be fine (also adding a comment a few lines above).

I am planning to leave this as 'async' unless there's a good reason not to.